docs/AnalyseTechnologique.md

```diff
@@ -0,0 +1,34 @@
+This document describes a comparison of technologies to develop a Desktop application on Windows which will lead to a choice to develop the project. Only technologies based on dotNet will be considered here.
```

Comment:
Maybe change the file name like `000-technical-analysis` ?

Reply:
Done